server/tsconfig.json

```diff
@@ -0,0 +1,103 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig to read more about this file */
+
+    /* Projects */
+    // "incremental": true,                              /* Save .tsbuildinfo files to allow for incremental compilation of projects. */
+    // "composite": true,                                /* Enable constraints that allow a TypeScript project to be used with project references. */
```

Comment:
remove commented lines,
add outDir, as this will be the output directory we need to run on production,
Also, we want to tell ts to exclude tests file from compiling by `exclude` attribute